leap.py

```diff
@@ -1,13 +1,14 @@
-# Function definition is here
-def is_leap_year( year ):
-   if (year%4 == 0 and year%100 != 0) or year%400 ==0:
-       print str(year) + " is a leap year"
-   else:
-        print str(year) + " is NOT a leap year"
-   return;
+def is_leap_year(year):
+    if (year%4 == 0 and year%100 != 0) or year%400 == 0:
+#https://www.python.org/dev/peps/pep-0008/#indentation
```

Comment:
@zarniak Code comments have the same level of indentation as the code they comment.